src/spyglass/utils/dj_mixin.py

```diff
@@ -131,6 +137,11 @@ def _safe_context(cls):
             else nullcontext()
         )
 
+    @property
+    def parallel_make(self):
+        """If table is parallelized in make function, override this function with True."""
+        return False
+
```

Comment:
I suggest tables use `_parallel_make = True` after the definition, like they do with `_nwb_table`

```suggestion
```